docs/release-notes/unreleased/issue-603-inventory-ui.md

```
@@ -0,0 +1,3 @@
+### Fixed
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)